### ISSUE TITLE
COM-visible classes with parameterized constructors should be noncreatable

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#411]
Given the code:
```
Public Class Example
    Friend Sub New(Bar As String)
        
    End Sub
End Class
```

tB will raise an error asking for a explicit constructor. However, I think the intention here is that it should be a public but non-creatable class. Adding `Attribute VB_Creatable = False` does not suppress the error. Not sure if we have an attribute implemented for `NonCreatable` as I don't see it in the intellisense list for attributes, either. But even without those explicit markers, I think such class without a non-parameterized class should be just treated as if it was non-creatable.

**Comment by WaynePhillipsEA** [comment]
Thanks @bclothier, yes that sounds sensible.    I'll add a `[ NotCreatable ]` attribute, and turn this error into a warning that gets suppressed with the NotCreatable attribute or the old style VB_Creatable.

**Comment by WaynePhillipsEA** [comment]
@bclothier I've fixed this in v0.10.5711.  Use the `[COMCtreatable(False)]` attribute.  Thanks
